CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+0.6.0-BETA / 2022-08-02
+===================
+In this version, the plugin API has been fully redesigned. The new API allows you to configure Kover in a more flexible manner, there is no need to configure Kover or test tasks separately.
```

Comment:
Please leave a link to a migration guide. Also, please ensure that link from deprecation message leads here

Reply:
https://github.com/Kotlin/kotlinx-kover/blob/v0.6.0-Beta/docs/migration-to-0.6.0.md accessible (temporary tag leads to this PR, then it will be placed on release commit)